### PR TITLE
Image signing

### PR DESCRIPTION
**Pull request by joe-braley**
Adds `az cli` command to trigger the image signing process.

### CODE REVIEW

File: .devops/build.yml [hunk 1]
```diff
@@ -249,3 +249,13 @@ extends:
...
+
+
+              - task: AzureCLI@2
+                displayName: Image signing
+                inputs:
+                  azureSubscription: "JEG-Infrastructure"
+                  scriptType: "bash"
+                  scriptLocation: "inlineScript"
+                  inlineScript: |
+                    az pipelines run --branch main --org ${{ parameters.organization }} --name $(OPENJDK_SIGNING)
```
Comment (jmjaffe37) [inline .devops/build.yml:261]: Nit, would prefer this stylistically:
```
az pipelines run \
  --branch main \
  --org ${{ parameters.organization }} \
  --name $(OPENJDK_SIGNING)
```

Comment (joe-braley) [reply .devops/build.yml:261]: Ha! I need to write this down so I don't forget ;)

File: .devops/build.yml [hunk 2]
```diff
@@ -249,3 +249,16 @@ extends:
                   DISTRIBUTION: $(distro)
                   INSTALLER_IMAGE: $(installer_image)
                   INSTALLER_TAG: $(installer_tag)
+
```
Comment (karianna) [inline .devops/build.yml:252]:
```suggestion
```

**Review by karianna — COMMENTED** [review]
I assume there's no official signing task or that the pipeline we run uses a signing task?

**Comment by joe-braley** [comment]
> I assume there's no official signing task or that the pipeline we run uses a signing task?

Yes, we trigger another workflow where the signing takes place.

**Review by jmjaffe37 — APPROVED** [review]

File: .devops/build.yml (hunk 1 above)
[the remark this answers is not in the document]
Comment (joe-braley) [reply .devops/build.yml]: I agree with using id. I will push the changes.

**Pull request merged**